apps/docs/src/common/pages/styles/images.tsx

```diff
@@ -48,41 +48,34 @@ const Page: FC<PageProps> = ({ location }) => (
         <li>how to find a reference number on a document complex.</li>
       </ul>
 
-      <p className="govuk-body"><A href="#Photography">Photography</A> can add visual interest to a page. We use it mostly in internal pages and services that use independent branding.</p>
-      <p className="govuk-body">Images must be relevant to the content, help users understand what they need to do and follow our guidance on the use of <A href="#Alternative text">Alternative text</A>.</p>
+      <p className="govuk-body"><A href="#Photography">Photography</A> can add visual interest to a page. We use it mostly in internal pages and services.</p>
+      <p className="govuk-body">Images must be relevant to the content, help users understand what they need to do and follow our guidance on the use of <A href="#Alternative text">alternative text</A>.</p>
 
       <h3 className="govuk-heading-m">Sourcing images</h3>
-      <p>Please contact the design system working group (<A href="mailto:design@digital.homeoffice.gov.uk">design@digital.homeoffice.gov.uk</A>) for advice on image use, for original source files and to commission illustrations.</p>
+      <p>Please contact <A href="mailto:design@digital.homeoffice.gov.uk">design@digital.homeoffice.gov.uk</A> for advice on image use and for original source files.</p>
+      <p>This helps us keep a record of what images we have.</p>
 
-      <h2 id="Illustrations or representative imagery" className="govuk-heading-l">Illustrations</h2>
+      <h2 id="Illustrations or representative imagery" className="govuk-heading-l">Illustrations or representative imagery</h2>
+      <p className="govuk-body">Illustrations must have a consistent style to create trust. This includes print products as well.</p>
       <p className="govuk-body">If your image represents something physical, such as a document, you should use the aspect ratio of that object.</p>
-      <p className="govuk-body">Illustrations must have a consistent style to create trust across online and print.</p>
+
 
       <div className="example example-images confirmation" style={{marginBottom: "2em"}}>
         <h2 className="govuk-heading-l">Do you have biometric residence card or permit?</h2>
         <img src={applyBrpExample} className="confirm-desktop" alt="An example of an illustrative image of a biometric residence permit card" />
         <span className="image-caption"><A href="https://www.gov.uk/employee-immigration-employment-status">Right to work</A></span>
       </div>
 
-      <div className="example example-images confirmation" style={{marginBottom: "2em"}}>
-        <img src={applyPassportIssue} className="confirm-desktop" alt="An example of an illustrative image of a passport and signposting where to find it's issue date." />
```

Comment:
Is there something at the top that should now go? i.e. `applyPassportIssue`?

Reply:
Yes, we got rid of one of the examples-  passport example. Is this causing an issue?

Reply:
Deleted the top code referencing an unused Passport image now